ETWProviders/etwprof.cpp

```diff
@@ -120,28 +120,35 @@ class CETWRegister
 } g_ETWRegister;
 
 // Redirector function for EventRegister. Called by macros in ETWProviderGenerated.h
-ULONG EVNTAPI EventRegister( LPCGUID ProviderId, PENABLECALLBACK EnableCallback, PVOID CallbackContext, PREGHANDLE RegHandle )
+#pragma warning(suppress: 28253)

```

Comment:
It is a good policy to *never* put in a warning suppression #pragma without explaining why. The simplest way to do that is with a comment that includes the warning text, 'cause who is going to remember what 28253 is?